docs/.vuepress/config.js

```diff
@@ -69,6 +69,9 @@ module.exports = {
       src: 'https://consent.cookiebot.com/uc.js',
       'data-cbid': 'ef171f1d-a288-433f-b680-3cdbdebd5646'
     }],
+    ['script', {}, `\
+var DOCS_VERSION = '${docsBase}';
```

Comment:
```suggestion
var DOCS_VERSION = '${getThisDocsVersion()}';
```
`docsBase` for the latest version point to the `"latest"` value

Reply:
True!